src/graphics/device.js

```diff
@@ -144,6 +144,12 @@ Object.assign(pc, function () {
      * @type {number}
      * @description The maximum supported texture anisotropy setting.
      */
+    /**
+     * @readonly
+     * @name pc.GraphicsDevice#extInstancing
+     * @type {boolean}
```

Comment:
Actually, this isn't necessarily a boolean. For WebGL 2 it's a boolean. For WebGL 1, it's an object. This should probably be tidied up in the engine code at some point. BTW, we don't document the other pc.GraphicsDevice#extXYZ properties. Maybe we could introduce something like:

```
this.supportsInstancing = !!this.extInstancing;
```

There's already precedent for that with `supportsMsaa` and `supportsStencil`.